spec/requests/omniauth_callbacks_controller_spec.rb

```diff
@@ -33,6 +33,39 @@
       expect(Users::OmniauthCallbacksController.find_authenticator("twitter"))
         .not_to eq(nil)
     end
+
+    context "with a plugin-contributed auth provider" do
+      before do
+        @provider = Plugin::AuthProvider.new
```

Comment:
we prefer the `let` syntax to using instance variables in tests.

Reply:
Done 👍